src/Components/Layout/NavBar.js

```diff
@@ -19,43 +20,74 @@ class NavBar extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      Navlink: ["Home", "Features", "Price", "Design"]
+      Navlink: [
+        {
+          name: "Home",
+          link: "/",
+        },
+        {
+          name: "Pricing",
+          link: "/pricing",
+        },
+      ],
     };
   }
 
   render() {
     // maps the links to be displayed at the top, eventually one of them can be like "Hello " + this.props.studentUsername
-    let navlinks = this.state.Navlink.map(item => <Nav.Link>{item}</Nav.Link>);
+    let navlinks = this.state.Navlink.map((item) => (
+      <Nav.Link>
+        <Link className="whiteLink" to={item.link}>
+          {item.name}
+        </Link>
+      </Nav.Link>
+    ));
     return (
       <div>
         <Navbar bg="dark" variant="dark" expand="md">
           <Navbar.Brand>TrackIT</Navbar.Brand>
           <Navbar.Toggle aria-controls="basic-navbar-nav" />
           <Navbar.Collapse id="basic-navbar-nav">
-            <Nav className="mr-auto">{navlinks}</Nav>
+            <Nav className="mr-auto">
+              {navlinks}
+              <Nav.Link>
+                <a
+                  className="whiteLink"
+                  href="https://github.com/acmutd/TrackIT/blob/master/README.md"
+                >
+                  Docs
+                </a>
+              </Nav.Link>
+            </Nav>
 
             <Nav className="ml-auto">
               {/* className="whiteStyle" will set the color to white */}
               <Nav.Link>
-                <FontAwesomeIcon
-                  className="whiteStyle"
-                  icon={faFacebook}
-                  size="lg"
-                />
+                <a href="https://github.com/acmutd/TrackIT">
+                  <FontAwesomeIcon
+                    className="whiteStyle"
+                    icon={faGithub}
+                    size="lg"
+                  />
+                </a>
               </Nav.Link>
               <Nav.Link>
-                <FontAwesomeIcon
-                  className="whiteStyle"
-                  icon={faGithub}
-                  size="lg"
-                />
+                <a href="https://www.instagram.com/utdacm/">
+                  <FontAwesomeIcon
+                    className="whiteStyle"
+                    icon={faInstagram}
+                    size="lg"
+                  />
+                </a>
               </Nav.Link>
               <Nav.Link>
-                <FontAwesomeIcon
-                  className="whiteStyle"
-                  icon={faLinkedin}
-                  size="lg"
-                />
+                <a href="https://www.acmutd.co/">
+                  <FontAwesomeIcon
+                    className="whiteStyle"
+                    icon={faCreativeCommonsBy}
```

Comment:
Can we add the acm logo here? Original asset should be in the other repos in the acm organization

Reply:
I found the `png` logo but it is quite difficult to incorporate in the navbar. The way I am thinking of includes editing the source code for the `fortawesome` library we are using.

Reply:
Lets not do that then, I'll work on acquiring an svg after mvp